README.md

```diff
@@ -167,12 +167,16 @@ Currently, these rules can cause linting failures:
 2. EXTRA_DOCUMENTED_COLUMNS: The predicted schema of the model does not have this column that was specified in the
    metadata
 
-### Usage with dbt-external-tables
+### Usage with dbt-external-tables and Column Mapping
```

Comment:
Don't think the `and Column Mapping` is needed here as it is part of the `Usage`

Reply:
Thanks, just rewrote the commit and removed that part.